stats/stats.go

```diff
@@ -110,52 +100,27 @@ func (s *StatsHandler) Handle(key string, initialMessage string, incomingMessage
 		time.Sleep(1 * time.Second)
 		count = 1
 	}
-
 	return
 }
 
 func writeStats(info *info.ContainerInfo, memLimit int64, writer io.Writer) error {
 	for _, stat := range info.Stats {
-		stat.Memory.Limit = uint64(memLimit)
 		data, err := json.Marshal(stat)
 		if err != nil {
 			return err
 		}
 
-		writer.Write(data)
-		writer.Write([]byte("\n"))
-	}
-
-	return nil
-}
-
-func resolveContainer(id string) (string, error) {
-	if id == "" {
-		return "", nil
-	}
-
-	client, err := dockerClient.NewClient(config.Config.DockerUrl)
-	if err != nil {
-		return "", err
-	}
-
-	container, err := client.InspectContainer(id)
-	if err != nil || container == nil {
-		return "", err
-	}
-
-	if useSystemd() {
-		return fmt.Sprintf("system.slice/docker-%s.scope", container.ID), nil
-	} else {
-		return fmt.Sprintf("docker/%s", container.ID), nil
-	}
-}
+		//because error was not being returned at this point, the for loop using this function never exits.
```

Comment:
Can you either remove or reword this comment? It's a good explanation of the bug you fixed, but I don't think it belongs in here long term. It won't make sense to someone looking at the code wwho doesnt have the context of the bug.